app/templates/main/volunteerProfile.html

```diff
@@ -122,7 +122,7 @@ <h2>Which programs do you want to make {{volunteer.firstName}} {{volunteer.lastN
                   {% set action = "remove" %}
                   {% set checked = "checked" %}
             {% endif %}
-                  <input  type="checkbox" id="mgr-{{ program.id }}" data-programid="{{program.id}}" {{checked}}  onclick="updateManagers(this,'{{volunteer.username}}','{{action}}')">
+                  <input class="program_checkboxes" type="checkbox" id="mgr-{{ program.id }}" data-programid="{{program.id}}" {{checked}}  onclick="updateManagers(this,'{{volunteer.username}}','{{action}}')">
```

Comment:
I don't see where you define or reference this class. If you do still need it, it should be singular, and use a dash instead of an underscore: `program-checkbox`